scripts/globals/spells/bluemagic/pinecone_bomb.lua

```diff
@@ -0,0 +1,65 @@
+-----------------------------------------
+-- Spell: Pinecone Bomb
+-- Additional effect: Sleep. Duration of effect varies with TP
+-- Spell cost: 48 MP
+-- Monster Type: Plantoids
+-- Spell Type: Physical (Piercing)
+-- Blue Magic Points: 2
+-- Stat Bonus: STR+1
+-- Level: 36
+-- Casting Time: 2.5 seconds
+-- Recast Time: 26.5 seconds
+-- Skillchain Element(s): Fire (can open Scission or Fusion and can close Liquefaction)
+-- Combos: None
+-----------------------------------------
+require("scripts/globals/bluemagic")
+require("scripts/globals/status")
+require("scripts/globals/magic")
+-----------------------------------------
+
+function onMagicCastingCheck(caster,target,spell)
+    return 0
+end
+
+function onSpellCast(caster,target,spell)
+    local params = {}
+    -- This data should match information on http://wiki.ffxiclopedia.org/wiki/Calculating_Blue_Magic_Damage
+    params.tpmod = TPMOD_DURATION
+    params.dmgtype = DMGTYPE_PIERCE
```

Comment:
Need to update this to `damageType` after https://github.com/project-topaz/topaz/commit/b4ed738e8901e0127ffe19b855d6a886f82c476e

Reply:
Done.